rtt/transports/mqueue/binary_data_archive.hpp

```diff
@@ -85,6 +85,13 @@
 #include <boost/serialization/item_version_type.hpp>
 #endif
 
+#if BOOST_VERSION >= 105900
```

Comment:
Maybe we can replace this check by
```cpp
#ifndef BOOST_PFTO
```
? That way there is no error if the macro would already be defined elsewhere, e.g. if the `boost/ptfo.hpp` is included elsewhere.